test/evergreen/configure_combinations.sh

```diff
@@ -37,7 +37,9 @@ echo `pwd`
 
 curdir=`pwd`
 
-flags="-DCMAKE_TOOLCHAIN_FILE=$curdir/cmake/toolchains/gcc.cmake -DCMAKE_C_FLAGS=\"-Werror -Wall -Wextra -Waddress -Waggregate-return -Wbad-function-cast -Wcast-align -Wdeclaration-after-statement -Wformat-security -Wformat-nonliteral -Wformat=2 -Wmissing-declarations -Wmissing-field-initializers -Wmissing-prototypes -Wnested-externs -Wno-unused-parameter -Wpointer-arith -Wredundant-decls -Wshadow -Wundef -Wunused -Wwrite-strings -O -fno-strict-aliasing -Wuninitialized\"
+# FIXME-WT-11788: Remove the -Wno-reserved-identifier flag once the violation of the standard
+# C11 7.1.3 has been addressed.
+flags="-DCMAKE_TOOLCHAIN_FILE=$curdir/cmake/toolchains/gcc.cmake -DCMAKE_C_FLAGS=\"-Werror -Wall -Wextra -Waddress -Waggregate-return -Wbad-function-cast -Wcast-align -Wdeclaration-after-statement -Wformat-security -Wformat-nonliteral -Wformat=2 -Wmissing-declarations -Wmissing-field-initializers -Wmissing-prototypes -Wnested-externs -Wno-reserved-identifier -Wno-unused-parameter -Wpointer-arith -Wredundant-decls -Wshadow -Wundef -Wunused -Wwrite-strings -O -fno-strict-aliasing -Wuninitialized\"
```

Comment:
Something to consider: afaict in CMake we can pull in external file containing the list of flags, potentially singular source for cmake and this script.
https://cmake.org/cmake/help/v3.11/command/file.html

Reply:
I don't mind the idea but I think it's out of scope for this PR. I can make a ticket if you like?

Reply:
Made WT-12965.

Reply:
thank you